docs/Installation.md

```diff
@@ -46,6 +46,14 @@ Follow the steps for your platform below.
 
 1. Prepare the configuration according to [this](Configuration.md) document and save it in `config.yaml`.
 
+    > **IMPORTANT NOTE**:
+    > It is recommended to limit access to the configuration file as it contains sensitive information.
+    > You can change access permissions to the configuration file using:
+    >
+    > ```bash
+    > chmod 640 config.yaml
+    > ```
+
```

Comment:
I wonder if we couldn't reuse this step as it's basically the same for all flavors/OSs below 🤔